### PR TITLE
Infobulle games desktop

### PR DESCRIPTION
**Pull request by Marionptnt**

### CODE REVIEW

File: public/assets/css/style.css [hunk 1]
```diff
@@ -113,9 +113,9 @@ button {
 .svg {
     position: fixed;
     z-index: 6;
-    transform: scaleY(0.8);
     margin-left: 12rem;
-    margin-top: 2rem;
+    margin-top: 4%;
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:117]: use rem for margin

File: public/assets/css/style.css [hunk 2]
```diff
@@ -113,9 +113,9 @@ button {
 .svg {
     position: fixed;
     z-index: 6;
-    transform: scaleY(0.8);
     margin-left: 12rem;
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:116]: no more than 4rem for margin if is higher, use flexbox or something else

File: public/assets/css/style.css [hunk 3]
```diff
@@ -733,54 +734,56 @@ button {
         max-width: 30vw;
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:734]: use px or % for width

File: public/assets/css/style.css [hunk 4]
```diff
@@ -733,54 +734,56 @@ button {
...
         align-self: center;
     }
-
-    .agencement-page-play, .agencement-page-cards-list{
-        display : flex;
+    .rules-play-input:checked+.rules-play-container {
+        clip-path: circle(80% at calc(100% - 3.5rem) 2.5rem);
+    }
+    .rules-text {
+        padding-left: 39rem;
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:762]: no more than 4rem for padding and margin, if is it higher, use flexbox or something else

File: public/assets/css/style.css [hunk 5]
```diff
@@ -733,54 +734,56 @@ button {
...
+        padding-right: 6rem;
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:763]: same here

**Review by MarjorieThomassin — CHANGES_REQUESTED** [review]

File: public/assets/css/style.css (hunk 3 above)
Comment (Marionptnt) [reply public/assets/css/style.css:734]: not my part

File: public/assets/css/style.css [hunk 6]
```diff
@@ -115,9 +115,9 @@ button {
 .svg {
     position: fixed;
     z-index: 6;
-    transform: scaleY(0.8);
-    margin-left: 12rem;
     margin-top: 2rem;
+    transform: scale(-0.6, 0.6);
+    transform: translateX(170px);
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:120]: what is the rational behind the value ? 
ps : the rational cannot be "I saw it on my screen" ;)

File: public/assets/css/style.css [hunk 7]
```diff
@@ -710,6 +711,9 @@ button {
 }
 
 @media screen and (min-width: 680px) {
+    .svg {
+        transform: translateX(300px);
```
Comment (MarjorieThomassin) [inline public/assets/css/style.css:715]: what is the rational behind the value ? 
ps : the rational cannot be "I saw it on my screen" ;)

**Review by MarjorieThomassin — CHANGES_REQUESTED** [review]

**Pull request merged**